uber/site_sections/mivs_admin.py

```diff
@@ -110,7 +110,7 @@ def checklist_completion(self, out, session):
     def discussion_group_emails(self, out, session):
         emails = []
         for studio in session.query(IndieStudio).join(IndieStudio.group).join(Group.guest):
-            emails.append(studio.group.guest.email)
+            emails.extend(studio.group.guest.email)
```

Comment:
Fixes a bug where this export was sometimes printing lists instead of the lists' elements, resulting in seemingly-random `['` and `']` pairs.